### PR TITLE
Added dogma

### PR DESCRIPTION
**Pull request by mintcore**

### CODE REVIEW

**Comment by braver** [comment]
@mintcore If you could resolve the conflict I'll merge it. You may want to check your readme for some placeholder stuff that needs removing though.

**Comment by mintcore** [comment]
Conflict resolved!

**Comment by braver** [comment]
The json is missing a `}` though

File: contrib.json [hunk 1]
```diff
@@ -234,6 +234,16 @@
...
+            "details": "https://github.com/mintcore/SublimeLinter-contrib-dogma",
+            "labels": ["linting", "SublimeLinter", "dogma", "elixir"],
+            "releases": [
+                {
+                    "sublime_text": ">=3000",
+                    "tags": true
+                }
+            ]
+        },
             "name": "SublimeLinter-contrib-dmd",
```
Comment (braver) [inline contrib.json]: Note that the json is invalid here

**Comment by mintcore** [comment]
Sorry, it was caused while trying the github editor. I recreated the branch based on the current master instead

**Pull request merged**